src/content/use-the-api/kubectl-gs/template-cluster.md

```diff
@@ -178,10 +193,38 @@ It supports the following flags:
 
+### vSphere specific flags
+
+- `--cluster-catalog` (optional) - Name of the Giant Swarm app catalog that holds the cluster's app release.
+- `--cluster-version` (optional) - Version of `cluster-vsphere` helm chart to use. If not provided, the latest version will be used.
+- `--default-apps-catalog` (optional) - Name of the Giant Swarm app catalog that holds the default-apps' app release.
+- `--default-apps-version` (optional) - Version of `default-apps-vsphere` helm chart to use. If not provided, the latest version will be used.
+- `--kubernetes-version` (optional) - Cluster's Kubernetes version (default: 1.24.11).
+- `--vsphere-control-plane-ip` (optional) - Control plane IP, leave empty for auto allocation.
+- `--vsphere-service-load-balancer-cidr` (optional) - CIDR for Service LB within workload cluster.
+- `--vsphere-network-name` - Portgroup name in vCenter to connect the new VMs to.
+- `--vsphere-control-plane-disk-gib` (optional) - Disk size in GiB for individual control plane nodes (default: 50).
+- `--vsphere-control-plane-ip-pool` (optional) - Name of `GlobalInClusterIpPool` CR from which to take an IP for the control plane (default: `wc-cp-ips`).
+- `--vsphere-control-plane-memory-mib` (optional) - Memory size in MiB for individual control plane nodes (default: 8096).
+- `--vsphere-control-plane-num-cpus` (optional) - Number of CPUs for individual control plane nodes (default: 4).
+- `--vsphere-control-plane-replicas` (optional) - Number of control plane replicas in odd number (default: 3).
+- `--vsphere-worker-disk-gib` (optional) - Disk size in GiB for individual worker nodes (default: 50).
+- `--vsphere-worker-memory-mib` (optional) - Memory size in MiB for individual worker nodes (default: 14144).
+- `--vsphere-worker-num-cpus` (optional) - Number of CPUs for individual worker nodes (default: 6).
+- `--vsphere-worker-replicas` (optional) - Number of worker replicas (default: 3).
+- `--vsphere-resource-pool` (optional) - Name of the vSphere resource pool to deploy the VMs to (default: cluster)
+- `--vsphere-image-template` (optional) - Name of the vSphere template to deploy the VMs from (default: `ubuntu-2004-kube-${kubernetes-version}`).
+- `--vsphere-credentials-secret-name` (optional) - Name of the kubernetes secret that should be associated to the cluster app. It should exist in the organization's namespace and should contain the credentials for vsphere.
+- `--oidc-issuer-url` (optional) - This is the issuer URL for configuring OpenID connect in the cluster API.
+- `--oidc-ca-file` (optional) - This is the CA file path in case is not used a trusted Certificate Authority for OIDC endpoint.
+- `--oidc-client-id` (optional) - This is the client ID that is configured in the OIDC endpoint.
+- `--oidc-username-claim` (optional) - This is the claim used to map the username identity of the user.
+- `--oidc-groups-claim` (optional) - This is the claim used to map the group identity of the user.
```

Comment:
Are these vSphere specifics?

Reply:
No but they were in the openstack section and I don't see them in the other providers sections. Maybe they're just not up to date. Also I don't know if they apply to vintage.
Do you think we should move them to the general cli flags?

Reply:
I do think so, IMO oidc should be a general set of params for all CAPI clusters

Reply:
WDYT https://github.com/giantswarm/docs/pull/1947/commits/4e3b37984f0dd0ed91ee44d2020a36e7cd14c7b7